wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/SchemaTest.kt

```diff
@@ -13,10 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress("UsePropertyAccessSyntax")
```

Comment:
It feels to me that this is not necessary. Correct me if I'm wrong.

Reply:
We want it otherwise we'll have warning for methods like `isNull()` or `isEmpty()`, etc.